apps/web/app/(routes)/project/page.tsx

```
@@ -0,0 +1,37 @@
+
+const TAB_COMPONENTS = {
+	overview: ProjectOverview,
+	updates: ProjectUpdatesSection,
+	impact: YourImpactSection,
+	showcase: ProjectShowcaseSection,
+} as const
+
+const ProjectDetailsPage = () => {
+	const tabs = projectTabsData.map((tab) => {
+		const Component = TAB_COMPONENTS[tab.id as keyof typeof TAB_COMPONENTS]
+		const contentComponent = Component ? <Component /> : null
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)